src/REPLMode/REPLMode.jl

```diff
@@ -509,7 +509,7 @@ function promptf()
     prefix = ""
     if project_file !== nothing
         if prev_project_file == project_file && prev_project_timestamp == mtime(project_file)
-            prefix = prev_prefix
+            prefix = prev_prefix::String
         else
             project_name = projname(project_file)
             if project_name !== nothing
```

Comment:
Is `project_name` guaranteed to be a `String` when it's not `nothing`? Just wondering about the safety of the typeassert (presumably, you're fine).

Reply:
I'm from what I can tell it should be.. the product of `string()` is always `String`, I believe??

Reply:
To play it safe I've reverted the type assert